core-models/src/main/java/com/anytypeio/anytype/core_models/NetworkMode.kt

```diff
@@ -0,0 +1,26 @@
+package com.anytypeio.anytype.core_models
+
+import com.anytypeio.anytype.core_models.NetworkModeConstants.NETWORK_MODE_CUSTOM
+import com.anytypeio.anytype.core_models.NetworkModeConstants.NETWORK_MODE_DEFAULT
+import com.anytypeio.anytype.core_models.NetworkModeConstants.NETWORK_MODE_LOCAL
+
+
+enum class NetworkMode(val value: String) {
+    DEFAULT(NETWORK_MODE_DEFAULT), LOCAL(NETWORK_MODE_LOCAL), CUSTOM(NETWORK_MODE_CUSTOM)
+}
+
+data class NetworkModeConfig(
+    val networkMode: NetworkMode = NetworkMode.DEFAULT,
+    val userFilePath: String? = null,
+    val storedFilePath: String? = null
+)
+
+object NetworkModeConstants {
+    const val NETWORK_MODE_PREF = "pref.network_mode"
```

Comment:
let's encapsulate these keys for shared preferences in `Persistence `module

Reply:
moved to NetworkModeProvider